Makefile

```diff
@@ -46,13 +47,16 @@ $(LOCAL_BRIDGE_PLUGIN_BINARY): $(SOURCES)
 generate: $(SOURCES)
 	go generate -x ./pkg/... ./plugins/...
 
-.PHONY: unit-test integration-test
+.PHONY: unit-test integration-test e2e-test
 unit-test: $(SOURCES)
 	go test -v -cover -race -timeout 10s ./pkg/... ./plugins/...
 
 integration-test: $(SOURCE)
 	go test -v -tags integration -race -timeout 10s ./pkg/... ./plugins/...
 
+e2e-test:  $(SOURCE) plugins
+	sudo -E CNI_PATH=${ROOT}/bin/plugins ${GO_EXECUTABLE} test -v -tags e2e -race -timeout 60s ./plugins/...
```

Comment:
This was the weirdness that I was running into:

```
[ec2-user@ip-1xx ~]$ go version
go version go1.7.6 linux/amd64
[ec2-user@ip-1xx ~]$ echo $PATH
/home/ec2-user/go176/go/bin:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/opt/aws/bin:/home/ec2-user/.local/bin:/home/ec2-user/bin
[ec2-user@ip-1xx ~]$ sudo -E go version
sudo: go: command not found
[ec2-user@ip-1xx ~]$ sudo -E echo $PATH
/home/ec2-user/go176/go/bin:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/opt/aws/bin:/home/ec2-user/.local/bin:/home/ec2-user/bin
```

@nmeyerhans  Any suggestions for alternatives here which can get rid of the need for `${GO_EXECUTABLE}`?